private/buf/bufworkspace/workspace_test.go

```diff
@@ -65,6 +65,7 @@ func testBasic(t *testing.T, subDirPath string) {
 		zap.NewNop(),
 		tracing.NopTracer,
 		bucket,
+		nil,
```

Comment:
It feels awkward to just pass a `nil`, should I create some `NewNopClientProvider` helper?

Reply:
yea